src/hotspot/share/jfr/support/jfrThreadLocal.hpp

```diff
@@ -33,6 +33,12 @@
 #include "runtime/atomic.hpp"
 #include "runtime/mutexLocker.hpp"
 
+#ifdef LINUX
```

Comment:
Can things now be fwd declared in the .hpp and includes move to the .cpp instead?

Reply:
Don't worry if it's too hard. It would just be a nice thing to eliminate these special includes from the header file.